docs/source/tutorials/pasha/pasha.md

```diff
@@ -1,36 +1,37 @@
-# PASHA: Efficient HPO with Progressive Resource Allocation
-Hyperparameter optimization (HPO) and neural architecture search (NAS) are methods of choice to obtain the best-in-class machine learning models, but in practice they can be costly to run. When models are trained on large datasets, tuning them with HPO or NAS rapidly becomes prohibitively expensive for practitioners, even when efficient multi-fidelity methods are employed. PASHA is an approach designed to tackle the challenge of tuning machine learning models trained on large datasets with limited computational resources. PASHA is able to dynamically allocate resources for the tuning procedure depending on the need. The experimental comparison shows that PASHA identifies well-performing hyperparameter configurations and architectures while consuming significantly fewer computational resources than solutions like ASHA.
+# PASHA: Efficient HPO and NAS with Progressive Resource Allocation
```

Comment:
Would you mind converting this into .rst? All our docs are .rst by now, I see no reason to have one using .md

Reply:
I've now converted this into .rst and removed the .md file